src/text-input-autocomplete.directive.ts

```diff
@@ -98,24 +100,43 @@ export class TextInputAutocompleteDirective implements OnDestroy {
   @HostListener('keypress', ['$event.key'])
   onKeypress(key: string) {
     if (key === this.triggerCharacter) {
+      this.usingShortcut = false;
       this.showMenu();
     }
   }
 
+  @HostListener('keydown', ['$event'])
+  onKeyDown(event: KeyboardEvent) {
+    // Binds to space and ctrlkey; keyCode and code are both used for broader compatibility
+    if ((event.keyCode === 32 || event.code === '32') && event.ctrlKey) {
```

Comment:
This won't work on a mac, as that uses the cmd key e.g.

```
const shortcutKey = /Mac/i.test(navigator.platform) ? 'metaKey' : 'ctrlKey';
```

Reply:
This won't be a problem anymore since the user now defines his own shortcuts. But just a small correction - it actually works on Macs as well. I did this on a Mac and the keys that I chose were based on a GUI for GraphQL, that has the same functionality. But I don't know if it it would break compatibility with older Macs (I am a relatively new Mac user). Thank you!